map_mutation_class.py

```diff
@@ -0,0 +1,51 @@
+"""
+Gregory Way 2017
+PanCancer Classifier
+map_mutation_class.py
+
+Usage: Run in command line with required command argument:
+
+        python map_mutation_class.py --scores $prediction_file --genes $genes
+
+prediction_file is a string and genes is a comma sep string of gene symbols
+
+Output:
+.tsv file of classifier scores according to mutation class
+"""
+
+import os
+import argparse
+import pandas as pd
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-s', '--scores',
+                    help='string of the location of classifier scores')
+parser.add_argument('-g', '--genes',
+                    help='string of the genes to extract')
+args = parser.parse_args()
+
+# Load command arguments
+prediction_file = os.path.join(args.scores, 'classifier_decisions.tsv')
+genes = args.genes.split(',')
+
+out_dir = os.path.join(os.path.dirname(prediction_file), 'tables')
+
+if not os.path.exists(out_dir):
+    os.makedirs(out_dir)
+
+out_file = os.path.join(out_dir, 'mutation_classification_scores.tsv')
+
+raw_mut_file = os.path.join('data', 'raw', 'mc3.v0.2.8.PUBLIC.maf')
```

Comment:
Do you want to define "mc3.v0.2.8.PUBLIC.maf" as a variable or this file name is fixed?